### PR TITLE
Trev8939/webmap keyword

### PR DESCRIPTION
**Pull request by TADraeseke**
Hi @priy1289 - Could I ask you to please review another sample for me? Thanks!

### CODE REVIEW

File: java/search-for-webmap/README.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# Search for webmap
+
+Find webmap portal items by using a search term.
+
+![Search for webmap App](search-for-webmap.png)
+
+## Use case
+
+Portals can contain many portal items and at times you may wish to query
+the the portal to find what you're looking for. In this example, we
```
Comment (priy1289) [inline java/search-for-webmap/README.md]: minor, 'the' used twice

File: java/search-for-webmap/src/main/java/com/esri/arcgisruntime/sample/searchforwebmap/MainActivity.java [hunk 2]
```diff
@@ -0,0 +1,231 @@
...
+        mPortalItemList = mPortalQueryResultSet.getResults();
+        PortalItemAdapter portalItemAdapter = new PortalItemAdapter(mPortalItemList,
+            portalItem -> addMap(mPortal, portalItem.getItemId()));
+        mRecyclerView.setAdapter(portalItemAdapter);
+        // open the drawer once there are results
+        mDrawer.openDrawer(Gravity.START);
+        // get 10 more results to fill the recycler view
+        getMoreResults();
+      } catch (Exception e) {
+        String error = "Error getting portal query result set: " + e;
```
Comment (priy1289) [inline java/search-for-webmap/src/main/java/com/esri/arcgisruntime/sample/searchforwebmap/MainActivity.java]: Probably we can show the custom error only in Toast and not the actual exception
" java.util.concurrent.ExecutionException: java.io.IOException: Loading of dependent object failed" 
as we are already logging it for logcat..

File: java/search-for-webmap/src/main/java/com/esri/arcgisruntime/sample/searchforwebmap/MainActivity.java [hunk 3]
```diff
@@ -0,0 +1,231 @@
...
+      if (mMapView.getMap().getLoadError() != null) {
+        String error = "Unable to load map: " + mMapView.getMap().getLoadError().getMessage();
+        Toast.makeText(this, error, Toast.LENGTH_LONG).show();
+        Log.e(TAG, error);
+      }
+    });
+  }
+
+  @Override
+  public boolean onCreateOptionsMenu(Menu menu) {
```
Comment (priy1289) [inline java/search-for-webmap/src/main/java/com/esri/arcgisruntime/sample/searchforwebmap/MainActivity.java]: I somehow don't see a menu in the app, the left arrow on top left of the app, opens up to an empty sidebar

**Comment by priy1289** [comment]
HI @TADraeseke  When I run the app, logcat throws error, "12765-12765/com.esri.arcgisruntime.sample.searchforwebmap E/RecyclerView: No adapter attached; skipping layout".

File: java/search-for-webmap/src/main/java/com/esri/arcgisruntime/sample/searchforwebmap/MainActivity.java (hunk 2 above)
Comment (TADraeseke) [reply java/search-for-webmap/src/main/java/com/esri/arcgisruntime/sample/searchforwebmap/MainActivity.java]: Oh good catch thanks. I've changed it to `e.getMessage()` which should now just report "Loading of dependent object failed"

**Comment by TADraeseke** [comment]
Hi @priy1289 - I've made those fixes now. I don't think its worth working around the recycler view adapter attached message, there's no data to add to it until the user has made a search of some kind of search.

**Pull request merged**